app/helpers/taxonomy_helper.rb

```diff
@@ -138,13 +139,22 @@ def show_resource_if_allowed(f, taxonomy, resource_options)
     ids = "#{association.where(nil).klass.to_s.underscore.singularize}_ids".to_sym
 
     content_tag(:div, :id => resource, :class => "tab-pane") do
-      all_checkbox(f, resource) +
-      multiple_selects(f, association.where(nil).klass.to_s.underscore.pluralize.to_sym, association, taxonomy.selected_or_inherited_ids[ids],
-        {:disabled => taxonomy.used_and_selected_or_inherited_ids[ids],
-         :label => translated_label(resource, :select)},
-        {'data-mismatches' => taxonomy.need_to_be_selected_ids[ids].to_json,
-         'data-inheriteds' => taxonomy.inherited_ids[ids].to_json,
-         'data-useds' => taxonomy.used_ids[ids].to_json })
+      attr = association.where(nil).klass.to_s.underscore.pluralize.to_sym
+      selected_ids = taxonomy.selected_or_inherited_ids[ids]
+      opts = options.merge({:disabled => taxonomy.used_and_selected_or_inherited_ids[ids], :label => translated_label(resource, :select)})
```

Comment:
this is needed only for old input.
```suggestion
      options[:label] = translated_label(resource, :select)
```

Reply:
Yeah but I still need it for `multiple_selects` method.